File: Day-02/Mini-Projects/java/NumberGuessingProgram.java
import java.util.Random;
import java.util.Scanner;

public class NumberGuessingProgram {
    public static void main(String[] args){

        Random random = new Random();
        Scanner scanner = new Scanner(System.in);

        int guess = 0;
        int attempts = 0;
        int min = 1;
        int max = 100;
        int randomNumber = random.nextInt(min,max +1);

        System.out.println("=====Number Guessing game==========");
        System.out.printf("Guess The Number Between %d-%d\n",min ,max);

        do {
            System.out.print("Enter the guess Number: ");
            guess = scanner.nextInt();
            attempts++;
            if (guess < randomNumber){
                System.out.println("TOO LOW! Try Again");
            } else if (guess > randomNumber) {
                System.out.println("TOO HIGH! Try Again");
            }else {
                System.out.println("\nCORRECT NUMBER WAS "+guess);
                System.out.println("Number of attempts "+attempts);
            }

        } while (randomNumber!= guess);

        scanner.close();

    }

}

File: Day-02/Mini-Projects/python/NumberGuessingProgram.py
import random

# Generate a random number between 1 and 100
min_num = 1
max_num = 100
random_number = random.randint(min_num, max_num)

guess = 0
attempts = 0

print("===== Number Guessing Game =====")
print(f"Guess the number between {min_num} and {max_num}")

while guess != random_number:
    guess = int(input("Enter your guess: "))
    attempts += 1

    if guess < random_number:
        print("TOO LOW! Try Again")
    elif guess > random_number:
        print("TOO HIGH! Try Again")
    else:
        print(f"\nCORRECT! The number was {guess}")
        print(f"Number of attempts: {attempts}")